src/modules/core/components/BannedTag/BannedTag.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { HTMLAttributes } from 'react';
+import { FormattedMessage, MessageDescriptor } from 'react-intl';
+
+import Icon from '~core/Icon';
+
+import styles from './BannedTag.css';
+
+const displayName = 'BannedTag';
+
+interface Props extends HTMLAttributes<HTMLSpanElement> {
+  text?: MessageDescriptor | string;
+  textValues?: { [key: string]: string };
```

Comment:
We have a defined type for this already, it's called `SimpleMessageValues`. Please use it:
```
import { SimpleMessageValues } from '~types/index';
```